packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -162,12 +161,6 @@ const OrderDetails = observer(() => {
             )}
             {!is_buy_order_for_user && (
                 <React.Fragment>
-                    <EmailVerificationModal
-                        email_address={order_store.user_email_address}
-                        is_email_verification_modal_open={order_store.is_email_verification_modal_open}
-                        onClickResendEmailButton={() => order_store.confirmOrderRequest(id)}
```

Comment:
Should we also refactor out modal props that receive handlers? I did not do it for rating modal since it would affect reusability

Reply:
For now we can since this modal is specific to orders only.